decam_process.py

```diff
@@ -207,158 +246,307 @@ def doIngest(repo, datafiles):
     config = IngestConfig()
     config.parse.retarget(DecamParseTask)
     # create an instance of the decam ingest task
-    ingestTask = ingest.DecamIngestTask(config=config)
+    IngestTask = ingest.DecamIngestTask(config=config)
```

Comment:
Is there a reason you changed this to capitalized? Capitalized names are for classes, whereas this is a class instance?